Vector_graphics_on_surfaces/doc/Vector_graphics_on_surfaces/PackageDescription.txt

```diff
@@ -0,0 +1,68 @@
+/// \defgroup PkgVGoS_Ref Vector Graphics on Triangulated Surface Meshes Reference
```

Comment:
I think we should find another name for the package.   It is not about Vector Graphics, but about geodesic paths.